central/auth/datastore/datastore_impl_test.go

```diff
@@ -0,0 +1,125 @@
+//go:build sql_integration
+
+package datastore
+
+import (
+	"context"
+	"testing"
+
+	pgStore "github.com/stackrox/rox/central/auth/store/postgres"
+	roleDataStore "github.com/stackrox/rox/central/role/datastore"
+	permissionSetPostgresStore "github.com/stackrox/rox/central/role/store/permissionset/postgres"
+	rolePostgresStore "github.com/stackrox/rox/central/role/store/role/postgres"
+	accessScopePostgresStore "github.com/stackrox/rox/central/role/store/simpleaccessscope/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/postgres/pgtest"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/uuid"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	testRole1 = "New-Admin"
+	testRole2 = "Super-Admin"
+	testRole3 = "Super Continuous Integration"
+)
+
+var (
+	existingRoles = set.NewFrozenStringSet(testRole1, testRole2, testRole3)
```

Comment:
[nit] The part `existing` may allude to some readers (me) the roles are provided by the system. Whereas they are created by this test.
I'd suggest to rename these to

```suggestion
	testRoles = set.NewFrozenStringSet(testRole1, testRole2, testRole3)
```